gems/smithy-cbor/spec/spec_helper.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+require 'simplecov'
+SimpleCov.start do
+  add_filter '/spec/'
+  add_filter 'gems/smithy/'
+  add_filter 'gems/smithy-client/'
+  add_filter 'gems/smithy-schema/'
+end
+
+require 'smithy-cbor'
+
+require 'smithy'
+require_relative '../../smithy-client/spec/support/client_helper'
```

Comment:
Later I think we can make a schema_helper and rely on it from smithy-schema to test the codecs.